src/.eslintrc.cjs

```diff
@@ -22,52 +23,54 @@ module.exports = {
         "quotes": [ "error", "double" ],
         "semi": [ "error", "always" ],
         "camelcase": "off",
-        // "no-unused-vars": "off",
         "eqeqeq": "off",
         "comma-dangle": [ "error", "never" ],
         "comma-style": [ "error", "last" ],
         "comma-spacing": "off",
-        "space-before-function-paren": [ "error", "never" ],
         "space-in-parens": [ "error", "always" ],
-        "keyword-spacing": [ "error", {
-            "overrides": {
-                "if": { "before": false, "after": false },
-                "else": { "before": true, "after": true },
-                "for": { "before": false, "after": false },
-                "while": { "before": false, "after": false }
-            }
-        } ],
         "space-before-blocks": [ "error", "always" ],
         "array-bracket-spacing": [ "error", "always" ],
         "object-curly-spacing": [ "error", "always" ],
         "space-unary-ops": "off",
         "spaced-comment": "off",
```

Comment:
Remove this rule, please

Reply:
`"space-unary-ops": "off"` and `"spaced-comment": "off"` removed